yb-voyager/src/query/queryissue/issues_perf.go

```diff
@@ -19,10 +19,21 @@ package queryissue
 import (
 	"fmt"
 
+	"github.com/samber/lo"
+
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/constants"
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/issue"
 )
 
+const (
+	COLUMN_NAME                  = "Column Name"
```

Comment:
It's a bit weird for map keys to have spaces like this. We should keep these lower case and underscore separated. 

At the html layer, we can convert these to the required format (capitalize each word + add spaces). I think @sanyamsinghal  had already done this for some other use-case.

Reply:
I thought of keeping the keys like this from our code itself as all the UI then can directly use it without having any conversion in between

Reply:
Okay understood, let me keep keys all lower case with underscore separated format.